base/ca/src/main/java/org/dogtagpki/server/ca/v2/AgentCertRequestACLFilter.java

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright Red Hat, Inc.
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+//
+package org.dogtagpki.server.ca.v2;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebFilter;
+
+import org.dogtagpki.server.v2.ACLFilter;
+
+@WebFilter(servletNames = "caCertRequest-agent")
+public class AgentCertRequestACLFilter extends ACLFilter {
+
+    private static final long serialVersionUID = 1L;
+
+    @Override
+    public void init() throws ServletException {
+        super.init();
```

Comment:
It's actually not necessary to call `super.init()`:
https://javaee.github.io/javaee-spec/javadocs/javax/servlet/GenericFilter.html#init--